tsconfig.json

```diff
@@ -22,5 +22,5 @@
     "target": "es6"
   },
   "include": ["./src/global.d.ts", "./src/**/*.ts*"],
-  "exclude": ["./node_modules", "./cypress", "./cypress.config.ts"]
+  "exclude": ["./node_modules", "./cypress", "./cypress.config.ts", "./playwright-ct.config.ts"]
```

Comment:
```suggestion
  "exclude": ["./node_modules", "./cypress", "./cypress.config.ts", "./playwright", "./playwright-ct.config.ts"]
```

Reply:
It looks like you're right, actually. I deliberately added `./playwright` and `./src/**/*.test-pw.ts*"` to only the `tsconfig-build` file because I thought with them excluded from tsconfig they wouldn't be type-checked when we do `npm run type-check`, but I've just tested it and it looks like they still are (which is a bit surprising to me, actually).